### PR TITLE
fix/fosa_verified

### PR DESCRIPTION
**Pull request by danielchuh**
Fix for FOSA user login

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -209,8 +209,10 @@ def create_superuser(self, username, password=None, email=None, **extra_fields):
     def auto_provision_user(self, **kwargs):
         # only auto-provision django user if registered as interactive user
         try:
+            username = kwargs['username']
+            # breakpoint()
```
Comment (Chrisdanyk) [inline core/models.py:213]: remove this

File: core/schema.py [hunk 2]
```diff
@@ -1336,6 +1336,7 @@ class Input(UserBase, OpenIMISMutation.Input):
     @classmethod
     def async_mutate(cls, user, **data):
         try:
+            print(f"...................... user {user}")
```
Comment (Chrisdanyk) [inline core/schema.py:1339]: Remove this

File: core/models.py (hunk 1 above)
Comment (Chrisdanyk) [inline core/models.py:213]: Remove this

File: core/schema.py (hunk 2 above)
Comment (Chrisdanyk) [inline core/schema.py:1339]: Remove this too

**Review by lepresk — CHANGES_REQUESTED** [review]
Just remove debug messages like Chris suggested

**Pull request merged**